### DISCUSSION TITLE
Windows version

### DISCUSSION THREAD

**Discussion by serchuz** [johnlindquist/kit#13] [Q&A]
Do you have a version for windows planned?
I think it would be useful to be able to install it with **npm** for those of us who use Windows.

Thanks a lot.

**Comment by johnlindquist** [comment]
It works with WSL on Windows:
https://docs.microsoft.com/en-us/windows/wsl/install-win10

(I use WSL whenever I do web dev on Windows)

It's not a traditional node package, so I'm not planning on using npm. I am looking into what getting it to work in the command prompt and/or powershell would take, but that won't happen untill sometime next year.

Thanks for checking it out 😊

**Reply by iamfredrickokereke** [reply]
Awesome, I would wait, thanks for doing this.
